### PR TITLE
Adding docs to repo for Gitbook sync

### PR DESCRIPTION
**Pull request by justinegeffen**
Adding docs to repo to sync with Gitbook.

### CODE REVIEW

**Comment by justinegeffen** [comment]
@hanzei, do you have admin access so we can sync this with Gitbook when it's approved? :)

File: docs/onboarding.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Onboarding Your Users
+
+When you’ve tested the plugin and confirmed it’s working, notify your team so they can connect their CircleCI account to Mattermost and get started. Copy and paste the text below, edit it to suit your requirements, and send it out.
+
+> Hi team,
+We've set up the Mattermost CircleCI plugin, so you can get notifications from CircleCI in Mattermost. To get started, run the `/circleci account connect` slash command from any channel within Mattermost to connect your Mattermost account with CircleCI. Then, take a look at the slash commands section for details about how to use the plugin.
```
Comment (nathanaelhoun) [inline docs/onboarding.md:6]: May we include a link to the gitbook docs with the "slash commands section" text?

File: docs/slash-commands.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# Slash Commands
+
+After your System Admin has configured the CircleCI plugin, run /circleci account connect in a Mattermost channel to connect your Mattermost and CircleCI accounts.
+By default, the commands use the project set by /circleci config, unless a specific project is specified by the argument `--project <vcs/org-name/project-name>` (possible on all commands).
```
Comment (nathanaelhoun) [inline docs/slash-commands.md]:
```suggestion
After your System Admin has configured the CircleCI plugin, run `/circleci account connect` in a Mattermost channel to connect your Mattermost and CircleCI accounts.
By default, the commands use the project set by `/circleci config`, unless a specific project is specified by the argument `--project <vcs/org-name/project-name>` (possible on all commands).
```

File: docs/webhooks-notifications.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# Webhooks Notificatons
+
+Subscribe a channel to notifications from a CircleCI project.
+
+1. In the channel you want to subscribe to notifications, type /circleci subscription add.
```
Comment (nathanaelhoun) [inline docs/webhooks-notifications.md]:
```suggestion
1. In the channel you want to subscribe to notifications, type `/circleci subscription add`.
```

**Review by nathanaelhoun — APPROVED** [review]
Thanks for taking care of this @justinegeffen :smile:

File: docs/onboarding.md (hunk 1 above)
Comment (justinegeffen) [reply docs/onboarding.md:6]: Good idea, we definitely can. :)

Comment (justinegeffen) [inline docs/onboarding.md:6]:
```suggestion
We've set up the Mattermost CircleCI plugin, so you can get notifications from CircleCI in Mattermost. To get started, run the `/circleci account connect` slash command from any channel within Mattermost to connect your Mattermost account with CircleCI. Then, take a look at the [slash commands section](https://mattermost.gitbook.io/plugin-circleci/slash-commands/) for details about how to use the plugin.
```

Comment (justinegeffen) [reply docs/onboarding.md:6]: Note: I need to validate the link once it's live.

**Review by nathanaelhoun — APPROVED** [review]

**Comment by hanzei** [comment]
> @hanzei, do you have admin access so we can sync this with Gitbook when it's approved? :)

@justinegeffen Not right now, but I'm trying to get admin access.

**Pull request merged**